modules/ols-configuring-custom-tls-certificates.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-custom-tls-certificates_{context}"]
+= Configuring custom TLS certificates
+
+Configure custom TLS certificates for secure {ols-long} service communication using certificates you generate, enabling centralized certificate management.
```

Comment:
centralized certificate management is just an example use case of custom TLS certificate. 
I think we can shorten the phrase to `Configure custom TLS certificates for secure {ols-long} service communication`